### PR TITLE
Feat/ip breadcrumb

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/solarsteein-dedicated`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#15708.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#15708.1 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#15708.2 ✘  
* 
  * Build ✘

File: packages/manager/apps/dedicated/client/app/ip/ip/antispam/ip-ip-antispam.controller.js [hunk 1]
```diff
@@ -1,122 +1,123 @@
...
-              reason,
-            );
-          },
-        );
-      };
+    $scope.hideAntispam = function hideAntispam() {
+      goToDashboard();
+      IpSpam.killPollSpamState();
+      // $rootScope.$broadcast('ips.display', 'table');
+      // $rootScope.$broadcast('ips.table.reload');
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/ip/ip/antispam/ip-ip-antispam.controller.js:71]:
```suggestion
```

**Review by JeremyDec — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build#15793.0 ✘  
* 
  * Build ✘

**Review by antleblanc — APPROVED** [review]

**Pull request merged**